src/System.Private.CoreLib/shared/System/Convert.cs

```diff
@@ -720,15 +721,15 @@ public static sbyte ToSByte(decimal value)
         }
 
         [CLSCompliant(false)]
-        public static sbyte ToSByte(string value)
+        public static sbyte ToSByte(string? value)
         {
             if (value == null)
                 return 0;
             return sbyte.Parse(value, CultureInfo.CurrentCulture);
         }
 
         [CLSCompliant(false)]
-        public static sbyte ToSByte(string value, IFormatProvider provider)
+        public static sbyte ToSByte(string value, IFormatProvider? provider)
```

Comment:
isn't value should be nullable?

Reply:
`sbyte.Parse` will throw an exception if `value` is null.  I don't know why it was designed this way, and I agree it's odd that it's not consistent with the others, but it is documented to throw in this case, and it's probably not worth changing, at least not as part of this exercise.
https://docs.microsoft.com/en-us/dotnet/api/system.convert.tosbyte?view=netframework-4.7.2#System_Convert_ToSByte_System_String_System_IFormatProvider_